scripts/relock.js

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env node
```

Comment:
If we have lockdown's hash in the thing, and it installs itself, so we can trust it, i think that would work. We definitely need at least lockdown.js in the repo.